front/app/components/FilterBoxes/StatusFilter.tsx

```diff
@@ -160,14 +158,23 @@ const StatusFilter = memo<Props>(
                   onClick={handleOnClick}
                   className={`e2e-status ${isFilterSelected ? 'selected' : ''}`}
                 >
-                  <T value={status.attributes.title_multiloc}>
-                    {(statusTitle) => <>{capitalize(statusTitle)}</>}
-                  </T>
+                  <Box display="flex">
+                    <ColorIndicator bgColor={status.attributes.color} />
+                    <Text
+                      m="0px"
+                      color={isFilterSelected ? 'white' : 'textPrimary'}
+                    >
+                      <T value={status.attributes.title_multiloc}>
+                        {(statusTitle) => <>{capitalize(statusTitle)}</>}
+                      </T>
+                    </Text>
+                  </Box>
```

Comment:
We could/should make one component of these. The benefit is limited now, but it would become apparent if we did more unit tests, for example.